src/App.js

```diff
@@ -27,23 +27,11 @@ const alex = {
 };
 
 const App = () => (
+  // eslint-disable-next-line
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person {...misha} />
```

Comment:
Don't use spread, because u don't know how props get into to component)

A better solution is to add the user (on any name😄 up to you) property
```suggestion
    <Person user={misha} />
```